.gitignore

```diff
@@ -1,3 +1,4 @@
 node_modules
 .env
 bot.log
+package-lock.json
```

Comment:
```suggestion
```
Actually there shouldn't be a `package-lock.json` at all, due to the project using Bun.